docs/computer_setup.md

```diff
@@ -2,82 +2,161 @@
 How to set up your system, environment, and upload file(s) into the database
 
 ## Setting up operating system
-Make sure you have an up-to-date web browser. Follow the directions [here](https://docs.microsoft.com/en-us/windows/wsl/install) to set up a bash shell, git, and python 3 (please set up all three). (add miniconda)
-* Run as administrator 
-	*  If using cmd, right-click on the program and click “Run as Administrator”
-* Skip distribution
+Make sure you have an up-to-date web browser. Follow the directions
+[here](https://docs.microsoft.com/en-us/windows/wsl/install) to set up a bash shell,
+git, and python 3. (add miniconda)
+- Run as administrator 
+	-  If using cmd, right-click on the program and click “Run as Administrator”
+- Skip distribution
 
 Here is some basic info about each of the programs: 
-* Bash shell
-	The bash shell is the interface between the user (you) and the operating system. It is a command line interface shell program extensively used in Linux and macOs operating systems. Bash is used when users want to control their computer without having to navigate menus, options, and windows while interacting in your computer. For example, if you want to quickly create, edit, or delete multiple files, it’s easier to use Bash instead of finding each file by pointing and clicking on multiple directories.
+- Bash shell
+	The bash shell is an interface between the user (you) and the operating system.
+	It is a command line interface extensively used in Linux and MacOSX operating
+	systems. Bash (and other command line interfaces) are used when users want to
+	control their computer programmatically. Command line interfaces are a much older
+	way to interact with computers and most people are much more familiar with the newer
+	graphical user interfaces (GUIs), which allow users to control their computer by
+	using menus, options, and windows. GUIs are a fantastic way for users to interact
+	with computers, but they do not work very well when we want to start automatting
+	complex tasks. If you want to do the same (complicated) thing to dozens or hundreds
+	of files it becomes slow, tedious and error prone to do that using a GUI. For those
+	kinds of tasks, a command line interface is much more appropriate. Command line
+	interfaces are interacted with exclusively via a keyboard (they were invented
+	before mice existed!):
 	
-	Basic command lines to interact with bash:
-	* ls: list directory contents. The ls command allows you to quickly view all files within a specified directory.
-	* pwd: print working directory you are in. If you want to know the exact directory you are working within, then pwd will tell you. 
-	* cd: change directory you are in.
-	* exit: exit out of the directory. This command will close a terminal window, and end the execution of a shell script. 
+	- Here are some commonly used commands in the Bash terminal:
```

Comment:
Consider introducing the format of space-separated 'command parameters...', along with terminology 'passing' paramereters to command and command 'returning' output to us.